Core/Utilities/Platform.py

```diff
@@ -68,18 +71,18 @@ def libc_ver( executable = sys.executable, lib = '', version = '',
       elif so:
         if lib != 'glibc':
           lib = 'libc'
-          version = max( version, soversion )
-          if threads and version[-len( threads ):] != threads:
+          version = max(version, [int(soversion)]).pop()
+          if threads and version[-len(threads):] != threads:
             version = version + threads
       pos = m.end()
-  return lib, '.'.join( map( str, version ) )
+  return lib, '.'.join(map(str, version))
```

Comment:
```suggestion
  return lib.decode(), '.'.join(map(str, version))
```
Also, I think the 5 strings containing `'glibc'`/`'libc'` should be bytes (`b'glibc'`/`b'libc'`) else they will never be equal in the comparisons.